README.md

```diff
@@ -82,6 +82,17 @@ There is also a demo application:
     cd tensorflow-mnist
     stack --docker --docker-image=$IMAGE_NAME build --exec Main
 
+If you want to use GPU you can do:
```

Comment:
Maybe add a header like

```markdown
### Docker GPU Support
```

so that it easier to scan through the docs.

Reply:
Sure thing